FreeRTOS-Plus/Source/Application-Protocols/network_transport/mbedtls_pk_pkcs11.c

```diff
@@ -73,15 +74,51 @@ typedef struct P11RsaCtx
     P11PkCtx_t xP11PkCtx;
 } P11RsaCtx_t;
 
+/*-----------------------------------------------------------*/
+
+/**
+ * @brief Allocates a P11EcDsaCtx_t
+ *
+ * @return A void pointer to the newly created P11EcDsaCtx_t.
+ */
 static void * p11_ecdsa_ctx_alloc( void );
 
+/**
+ * @brief Initializes a P11EcDsaCtx_t with the given PKCS11 parameters.
+ *
+ * @param pvCtx Void pointer to the P11EcDsaCtx_t.
+ * @param pxFunctionList Pointer to a CK_FUNCTION_LIST for the PKCS11 module.
+ * @param xSessionHandle An initialized CK_SESSION_HANDLE for the given PKCS11 module.
+ * @param xPkHandle The CK_OBJECT_HANDLE for the target private key.
+ * @return CK_RV
```

Comment:
Should the return comment be more explicit? I'm not sure what a CK_RV is, but if someone more familiar can understand what this means, then this can be left.